lib/paperclip/schema.rb

```diff
@@ -51,7 +51,7 @@ def attachment(*attachment_names)
         attachment_names.each do |attachment_name|
           COLUMNS.each_pair do |column_name, column_type|
             column_options = options.merge(options[column_name.to_sym] || {})
-            column("#{attachment_name}_#{column_name}", column_type, column_options)
+            column("#{attachment_name}_#{column_name}", column_type, **column_options)
```

Comment:
Did it break our CI or was it just failing on bundling?
Just curious how you find that has to be changed.

Reply:
It was failed during the `rake db:migrate` on CI, so I found a failed migration with attachement type field and understood that it comes from `paperclip`.